apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceExpectationsTest.kt

```diff
@@ -16,20 +18,24 @@ class CharSequenceExpectationsSpec : ch.tutteli.atrium.specs.integration.CharSeq
     fun1<CharSequence, Regex>(Expect<CharSequence>::toMatch),
     fun1<CharSequence, Regex>(Expect<CharSequence>::notToMatch)
```

Comment:
nitpick: if you remove the suppression below, could you please also remove these explicit type parameters. Looks like they are no longer necessary (Kotlin was improved since we had to add them :slightly_smiling_face: )
```suggestion
    fun1(Expect<CharSequence>::toMatch),
    fun1(Expect<CharSequence>::notToMatch)
```
same same for the other lines above and also in api-infix